packages/jobs/lib/execution/webhook.ts

```diff
@@ -43,6 +57,20 @@ export async function startWebhook(task: TaskWebhook): Promise<Result<void>> {
 
         const logCtx = await logContextGetter.get({ id: String(task.activityLogId) });
 
+        syncJob = await createSyncJob({
+            sync_id: sync.id,
+            type: SyncType.INCREMENTAL,
+            status: SyncStatus.RUNNING,
+            job_id: task.name,
+            nangoConnection: task.connection,
+            sync_config_id: syncConfig.id!,
+            run_id: task.id,
+            log_id: logCtx.id
+        });
```

Comment:
I checked, job table is not used for deduplication so it should not block syncs to run. But happy to get a second pair of eyes

Reply:
job table is not used by the scheduler, which also means sync and webhook scripts could run concurrently and create all sort of undefined behavior if saving the same model. I think we should prevent this scenario to happen at compile time to avoid future headaches

Reply:
it could be a legitimate use case though, let's say you have a sync for your customers and get a webhook at each new customer, it's handy to be able to push the new customer directly to the record list without waiting for the next sync to trigger.

Reply:
I know but nango is not fully designed for that at the moment